src/components/benefit-cta.vue

```diff
@@ -12,7 +12,8 @@
     <router-link
       v-if="
         benefit.institution.etablissements &&
-        benefit.institution.etablissements.length > 0
+        benefit.institution.etablissements.length > 0 &&
+        route !== routeAide
```

Comment:
```suggestion
        $route.name !== 'aide'
```
A tester mais normalement ca fonctionne et ça évite de changer quand l'url change